frontend/Dockerstage

```diff
@@ -4,4 +4,3 @@ ARG OFRAK_DIR=.
 COPY --chown=node:node $OFRAK_DIR/frontend /home/node/frontend
 
 WORKDIR /home/node/frontend
-RUN su node -c "npm install && npm run build"
```

Comment:
This command should stay in. Users building the Docker image will still need to build the front end.